tools/build_langserver/langserver/utils_test.go

```diff
@@ -192,6 +192,26 @@ func TestLooksLikeDictAttr(t *testing.T) {
 	assert.False(t, LooksLikeDictAttr("{\"foo\":2}.keys()"))
 }
 
+func TestExtractBuildLabel(t *testing.T) {
+	label := ExtractBuildLabel("target = \"//src/cache/blah:hello")
```

Comment:
FYI Go has raw strings that would be nicer here: 
```
`target = "//src/cache/blah:hello`
```

Reply:
nice!